Sources/Overload/OvEditor/src/OvEditor/Core/EditorRenderer.cpp

```diff
@@ -153,7 +153,7 @@ OvMaths::FMatrix4 OvEditor::Core::EditorRenderer::CalculateCameraModelMatrix(OvC
 {
 	auto translation = FMatrix4::Translation(p_actor.transform.GetWorldPosition());
 	auto rotation = FQuaternion::ToMatrix4(p_actor.transform.GetWorldRotation());
-	auto scale = FMatrix4::Scaling({ 0.4f, 0.4f, 0.4f });
+	auto scale = FMatrix4::Scaling({ 0.5f, 0.5f, 0.5f });
```

Comment:
Instead of scaling the internal fbx of the editor with downscaling matrix of 1/2, we should export them to the correct scale beforehand. That line is confusing for a reader in order to understand "why" those values instead of 1

Reply:
This will certainly be removed with the integration of the new no-scaled editor models.

Reply:
We discussed about it with Thibaut, and we think that we should pre-scale the model instead of manually scaling it in the `EditorRenderer`. I will update this ASAP